shared/syncer.js

```diff
@@ -24,7 +24,8 @@ if (isServer) {
   var serverOnly_qs = 'qs';
   var qs = require(serverOnly_qs);
 } else {
-  Backbone.$ = window.$ || require('jquery');
+  var $ = window.$ || require('jquery');
```

Comment:
ah, if it's for jshint then :+1: wasn't sure if you had any other reasons for doing it.